multisig/proposals/arbitrumone/vip-008/index.ts

```diff
@@ -0,0 +1,82 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { NETWORK_ADDRESSES, ZERO_ADDRESS } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { arbitrumone } = NETWORK_ADDRESSES;
+
+export const XVS_VAULT_PROXY = "0x8b79692AAB2822Be30a6382Eb04763A74752d5B4";
+export const XVS_STORE = "0x507D9923c954AAD8eC530ed8Dedb75bFc893Ec5e";
+export const XVS_VAULT_REWARDS_SPEED = "578703703703703"; // to be decided
```

Comment:
👍 
1,500 XVS/month, as it is proposed [here](https://community.venus.io/t/incentive-model-proposal-for-arbitrum-deployment/4408)